delete-review-app/action.yml

```diff
@@ -0,0 +1,140 @@
+name: Delete Review App
+description: Delete Review App
+inputs:
+  azure-credentials:
+    description: A JSON string containing service principal credentials.
+    required: true
+  gcp-wip:
+    description: The full identifier of the GCP Workload Identity Provider.
+    required: false
+  gcp-project-id:
+    description: The name of the GCP Project ID.
+    required: false
+  environment:
+    description: Environment to delete from
+    default: review
+    required: false
+  github-event-name:
+    description: github event
+    default: pull_request
+    required: false
+  global-config:
+    description: Environment for global config parameters
+    default: review
+    required: false
+  pr-number:
+    description: PR number for the review app
+    required: true
+  service-name:
+    description: name of the service
+    required: true
+  service-pretty:
+    description: pretty name of the service
+    required: true
+  service-short:
+    description: short name of the service
+    required: true
+  terraform-base:
+    description: Path to the terraform files
+    required: false
+    default: "terraform/application"
+  terraform-destroy-command:
+    description: terrafom destroy command in makefile
+    required: false
+    default: "terraform-destroy"
+
+runs:
+  using: composite
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Set terraform version
+      id: set_env_var
+      shell: bash
+      run: |
+        terraform_version=$(awk '/{/{f=/^terraform/;next}f' ${{ inputs.terraform-base }}/terraform.tf | grep -o [0-9\.]*)
+        echo "TERRAFORM_VERSION=$terraform_version" >> $GITHUB_ENV
+
+    - name: Use Terraform ${{ env.TERRAFORM_VERSION }}
+      uses: hashicorp/setup-terraform@v3
+      with:
+        terraform_version: ${{ env.TERRAFORM_VERSION }}
+        terraform_wrapper: false
+
+    - uses: azure/login@v2
+      with:
+        creds: ${{ inputs.azure-credentials }}
+
+    - uses: google-github-actions/auth@v2
+      if: ${{ inputs.gcp-wip != '' }}
+      with:
+        project_id: ${{ inputs.gcp-project-id }}
+        workload_identity_provider: ${{ inputs.gcp-wip }}
+
+    - uses: DFE-Digital/github-actions/set-kubelogin-environment@master
+      with:
+        azure-credentials: ${{ inputs.azure-credentials }}
+
+    - name: Terraform Destroy
+      shell: bash
+      run: |
+        make ci ${{ inputs.environment }}  ${{ inputs.terraform-destroy-command  }}
```

Comment:
replace inputs.terraform-destroy-command with terraform-destroy, and inputs.environment with review